js/net_test.ts

```diff
@@ -126,7 +131,9 @@ testPerm({ net: true }, async function netCloseWriteSuccess() {
   listener.close();
   conn.close();
 });
+*/
 
+/* TODO Fix broken test.
```

Comment:
How did these get broken?

Reply:
They just landed, I think they weren't ever working properly. I think it's actually how this bug was discovered.